playbooks/roles/vault/tasks/main.yml

```diff
@@ -9,7 +9,6 @@
   user:
     name: vault
     group: vault
-    createhome: no
```

Comment:
Is this intentional? This will change behavior because the default is `yes` - https://docs.ansible.com/ansible/2.5/modules/user_module.html.

Reply:
Yes, this is intentional. Without it vault -dev will fail to create ~/.vault-token.

Reply:
Technically it makes no difference, but it's generally unneeded. It might be confusing though because other roles don't create homes, so to be consistent we probably shouldn't create it, and you can change this with a flag.

Reply:
I'll do that to be consistent, good point.